lib/chelsea/formatters/text.rb

```diff
@@ -17,6 +18,7 @@ def get_results(server_response, reverse_dependencies)
       end
 
       i = 0
+      vuln_count = 0
```

Comment:
@gmohre I am ABSOLUTELY sure there is a better way to get this count, using like Rubyish things, but you know, I am not up to your Ruby-fu.